lib/rules/web/console/base/login.xyaml

```diff
@@ -15,15 +15,15 @@ login_sequence:
     if_element:
       selector: &pre_login_buttons
         # we just choose the first button per OPENSHIFTQ-12062
```

Comment:
@yapei seems like an outdated comment, pls remove it.

Reply:
ok, will do